src/Compilers/VisualBasic/Portable/LanguageVersion.vb

```diff
@@ -80,10 +80,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         <Extension>
         Public Function MapSpecifiedToEffectiveVersion(version As LanguageVersion) As LanguageVersion
             Select Case version
-                Case LanguageVersion.Latest
-                    Return LanguageVersion.VisualBasic15_5
-                Case LanguageVersion.Default
-                    Return LanguageVersion.VisualBasic15
+                Case LanguageVersion.Latest,
+                     LanguageVersion.Default
+                    Return LanguageVersion.VisualBasic16
```

Comment:
>VisualBasic16 [](start = 43, length = 13)

Let's confirm with Kathleen/Philip that the language version is "16" as opposed to "16.3".